complex/ft_comp.c

```diff
@@ -149,7 +149,6 @@ static int ft_comp_x(struct fid_cq *cq, struct ft_xcontrol *ft_x,
 		do {
 			if (!poll_time)
 				clock_gettime(CLOCK_MONOTONIC, &s);
-
```

Comment:
Please revert all changes to this file and try to avoid changes that add 'noise' to the actual changes submitted.